litex/boards/targets/versa_ecp5.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, platform, sys_clk_freq):
 # BaseSoC ------------------------------------------------------------------------------------------
 
 class BaseSoC(SoCSDRAM):
-    def __init__(self, sys_clk_freq=int(75e6), toolchain="diamond", **kwargs):
+    def __init__(self, sys_clk_freq=int(50e6), toolchain="diamond", **kwargs):
```

Comment:
Would it not be better to keep the original sys_clk_freq; and only switch down to 50MHz if using Rocket?

Reply:
Yes, i agree, i already reverted it: https://github.com/enjoy-digital/litex/commit/cf369c437c657eae0aa72ad5524ef37d9739f172 and the others changes that were introducing change to others targets.